relay-general/src/pii/convert.rs

```diff
@@ -1410,4 +1411,32 @@ THd+9FBxiHLGXNKhG/FRSyREXEt+NyYIf/0cyByc9tNksat794ddUqnLOg0vwSkv
         process_value(&mut data, &mut pii_processor, ProcessingState::root()).unwrap();
         assert_annotated_snapshot!(data);
     }
+
+    #[test]
+    fn test_scrub_object() {
+        let mut data = Event::from_value(
+            serde_json::json!({
+                "request": {
+                    "url": "https://www.bitumequebec.ca/fr/adhesions/authentication",
+                    "method": "POST",
+                    "data": {
+                      "password": "test",
+                      "profile": {
+                        "email": "testing@example.com",
+                        "password": {
+                          "$eq": "u{}{}{}]H[[[]]ww6KrA9F.x-F%%"
```

Comment:
Just to double check, was this copied from a real event?

Reply:
yes, see the linked issue. The password is just a test random string here.